src/mappings/modules/liquidation/dog.ts

```diff
@@ -66,3 +67,26 @@ export function handleFileClip(event: File3): void {
     ilk.save()
   }
 }
+
+export function handleBark(event: Bark): void {
+  let idStr = event.params.id.toString()
+  let vaultId = event.params.urn.toHexString() + '-' + event.params.ilk.toString()
+  let collateralTypeId = event.params.ilk.toString()
+  let saleAuction = saleAuctions.loadOrCreateSaleAuction(idStr, event)
+  saleAuction.vault = vaultId
+  saleAuction.collateralType = collateralTypeId
+  saleAuction.clipperContract = event.params.clip
```

Comment:
I guess there is no need to do this here. the clip is been updated on the File event